test/select_io_dset.c

```diff
@@ -127,7 +142,8 @@ test_no_type_conv(hid_t fid, unsigned chunked, unsigned dtrans, unsigned mwbuf)
     int         trans_wbuf[DSET_SELECT_DIM];
     int         rbuf[DSET_SELECT_DIM];
     char        dset_name[DSET_NAME_LEN];
-    const char *expr = "2*x";
+    const char *expr                  = "2*x";
+    uint32_t    no_selection_io_cause = 0;
```

Comment:
I forgot to remove the declaration (and subsequent ones) for "no_selection_io_cause".

Reply:
I'm planning to add the tests back in, needs more work though